src/libraries/SVGElements.sol

```diff
@@ -236,16 +236,18 @@ library SVGElements {
             return 0;
         }
 
-        unchecked {
-            uint256 charWidth = largeFont ? 16 : 13;
-            uint256 semicolonIndex;
-            for (uint256 i = 0; i < length; ++i) {
-                if (bytes(text)[i] == ";") {
-                    semicolonIndex = i;
-                }
+        uint256 charWidth = largeFont ? 16 : 13;
+        uint256 semicolonIndex;
+        for (uint256 i = 0; i < length; ++i) {
+            if (bytes(text)[i] == ";") {
+                semicolonIndex = i;
+            }
+            unchecked {
```

Comment:
why did you make this change? it should have remained the same

Reply:
In the original code, the `unchecked` block encompassed the arithmetic operations, ternary statement and the loop that checks for the semicolon character in the text. It might imply that the `unchecked` keyword is also optimizing the other expressions which are not arithmetics, which is not the case anymore. 

So by moving it closer to the arithmetic operations, my goal was to make it clear that it only applies to these operations and does not affect other operations.

Even though this change resulted in two `unchecked` statements instead of one. However, IMO the increased clarity and explicitness of the code outweighs the minor redundancy introduced by the additional `unchecked` statement.

What do you think?

Reply:
> keyword is also optimizing the other expressions which are not arithmetics, which is not the case anymore

i am not sure i understand what you mean, are you referring to the if statement? or to the loop counter?

> my goal was to make it clear that it only applies to these operations and does not affect other operations.

what other operations are "affected"?

the loop counters, are "affected" by default by an `unchecked` block

> IMO the increased clarity and explicitness of the code outweighs the minor redundancy introduced by the additional unchecked statement

i see. personally, i believe that having multiple `unchecked` blocks decreases clarity when reading the code, as it results in the code being nested within two blocks instead of one. imo, the redundancy outweighs the benefits.

Reply:
The point that I wanted to make is `unchecked` only impacts `width += charWidth` and `width -= charWidth * semicolonIndex` and not every expression that it covers. That's why I changed this code so it appears to be covering only these two expressions. 

> i believe that having multiple unchecked blocks decreases clarity when reading the code

I agree with you now. Having two unchecked statements in such a short block goes against DRY principle. I will update it now.

Reply:
Agree with @andreivladbrg.